client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -305,6 +308,8 @@ interface ChatClient {
 
     fun getVersion(): String
 
+    fun hasValidUrl(attachment: Attachment): Boolean
```

Comment:
Where is this method going to be called from? Does it have to exposed through `ChatClient` specifically?